app/assets/stylesheets/scaffolds.scss

```diff
@@ -6,7 +6,7 @@ body {
   background-color: #fff;
   color: #333;
   margin: 33px;
-  font-family: verdana, arial, helvetica, sans-serif;
+  font-family: arial, helvetica, sans-serif;
```

Comment:
I'm not sure what this changes in practice. I think we can just delete this file as it's not used (and was auto-generated by Rails)